lib/nx_zlib.h

```diff
@@ -569,6 +569,8 @@ extern int nx_deflateInit2_(z_streamp strm, int level, int method, int windowBit
 extern int nx_deflate(z_streamp strm, int flush);
 extern int nx_deflateEnd(z_streamp strm);
 extern unsigned long nx_deflateBound(z_streamp strm, unsigned long sourceLen);
+extern int nx_deflateSetDictionary(z_streamp strm, const unsigned char *dictionary,
+				uint  dictLength);
```

Comment:
Would be better to have this in the "Fix deflateSetDictionary and inflateSetDictionary" commit.

Reply:
Done.